src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -86,12 +86,16 @@ export class Feedback implements BaseEntity {
         return that.filter(Feedback.hasCreditsAndComment).length > 0 && that.filter(Feedback.hasCreditsAndComment).length === that.length;
     }
 
-    public static forModeling(credits: number, text?: string, referenceId?: string, referenceType?: string): Feedback {
+    public static forModeling(credits: number, text?: string, referenceId?: string, referenceType?: string, dropInfo?: any): Feedback {
```

Comment:
I think it would be nice if you define the type for `dropInfo` so that we know what it is (don't use `any`).